pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -566,6 +590,52 @@ def download(date_array, data_path, tag='', inst_id='', supported_tags=None,
                                   date.strftime('%d %B %Y'))))
         # Pause to avoid excessive pings to server
         sleep(0.2)
+
+    if zip_method:
+        # Cleanup temporary directory
+        temp_dir.cleanup()
+
+    return
+
+
+def get_file(remote_file, data_path, fname, temp_path=None, zip_method=None):
+    """Retrieve a file, unzipping if necessary.
+
+    Parameters
+    ----------
+    remote_file : file content
+        File content retireved via requests.
+    data_path : str
+        Path to pysat archival directory.
+    fname : str
+        Name of file on the remote server.
+    temp_path : str
```

Comment:
What happens is None is provided?

Reply:
A TypeError. D:

Reply:
If that's the case, shouldn't this be an arg?  If not, there should be a more informative error.

Reply:
This whole function should probably be replaced with `_get_file` as it is not intended to be directly invoked by a user